### ISSUE TITLE
when I use thonny to run multiprosess script,it's doesn't work

### ISSUE THREAD

**Issue by Dleta** [thonny/thonny#808]
```
#the script below 
import multiprocessing as mp
from time import sleep
def ab(x):
    while True:
         sleep(1)
         print(x)
if __name__ == '__main__':
    f1 = mp.Process(target = ab, args=('a'))
    f2 = mp.Process(target = ab, args=('b'))
    f1.start()
    f2.start()
    f1.join()
    f2.join()
```

**Comment by aivarannamaa** [comment]
There are syntax errors in your script...
Or do you get some other errors?

**Comment by Dleta** [comment]
the error is so simple,so you can fix i.,after you fixed it,you can find the script run in MS cmd with no problems,but have problems run in thonny and IDLE.
you can try it.

**Comment by Dleta** [comment]
I fixed it totally now.

**Comment by aivarannamaa** [comment]
I did fix and try it, but didn't notice any problems. Which problems do you mean?

**Comment by Dleta** [comment]
??????no problem?why i run it in idle or thonny is no reply?????i run it on windows10 platform,python3.6.8 64bit

**Comment by Dleta** [comment]
i should show you a picture,but i do not know how to.

**Comment by Dleta** [comment]
the script should print
a
b
a
b
a
.
.
.
but no reply on my computer

**Comment by aivarannamaa** [comment]
Hmm, you are right. I works when Thonny is run from my development environment, but stops working when it is installed.

I'll try to find a solution.

**Comment by Dleta** [comment]
thanks for you work

**Comment by aivarannamaa** [comment]
Thanks for the report!

The fix will be in the next beta of version 3.2.0 (to be published in couple of days)
